test/unit_tests/braket/circuits/test_observables.py

```diff
@@ -78,7 +78,7 @@ def test_basis_rotation_gates(
     "testobject,gateobject,expected_ir,basis_rotation_gates,eigenvalues", testdata
 )
 def test_eigenvalues(testobject, gateobject, expected_ir, basis_rotation_gates, eigenvalues):
```

Comment:
It'd be great if you could add a test case with self-defined Hermitian observables in tensor products as well, just to make sure nothing breaks.

Reply:
Will do